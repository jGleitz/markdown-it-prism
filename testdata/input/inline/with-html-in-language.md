`System.out.println("bar")`{language="><img/onerror="alert('hacked')"src=.><span/class="}
