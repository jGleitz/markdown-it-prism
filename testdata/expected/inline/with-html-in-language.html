<p><code class="language-&quot;&gt;&lt;img/onerror=&quot;alert('hacked')&quot;src=.&gt;&lt;span/class=&quot;">System.out.println(&quot;bar&quot;)</code></p>
